packages/transmute-cli/src/commands/provision/index.js

```diff
@@ -17,7 +17,13 @@ module.exports.minikube = (dryrun, clusterName, minikubeDriver) => {
     ' --memory ' +
     MINIKUBE_MEMORY;
   +' --profile ' + MINIKUBE_PROFILE;
-  let prov_cmd = minikube_start + ' --vm-driver=virtualbox';
+
+  if (minikubeDriver == 'none') {
+    minikube_start = 'sudo ' + minikube_start;
```

Comment:
Should we provide a message before this executes to let the user know why they are going to have to input their password?

Reply:
yes